nvi-rest/src/main/java/no/sikt/nva/nvi/utils/ExceptionMapper.java

```diff
@@ -35,7 +35,7 @@ public static <T> ApiGatewayException map(Failure<T> failure) {
             logger.error("IllegalStateException", exception);
             return new ConflictException(exception.getMessage());
         }
-        if (exception instanceof UnauthorizedOperationException) {
+        if (exception instanceof UnauthorizedOperationException || exception instanceof UnauthorizedException) {
```

Comment:
why not just throw an UnauthorizedOperationException since we already handle that one

Reply:
Because it's not a request to do an operation?